docs/source/quickstart.md

```diff
@@ -0,0 +1,8 @@
+# Quickstart
+
+```python
+python3 -m venv venv
+pip install "scipy<=1.7.3,>=0.16" "numpy<1.24,>=1.18"
```

Comment:
can we get rid of this?

Reply:
@sebastianneubauer  Still need to update the readme. For developing `poetry install` is enough to install regular/dev dependencies, `poetry install --with docs` lets you build the docs, `--with jupyter` gives you a jupyter lab etc. Don't forget to either activate the env with `poetry shell` or prepend your commands (e.g., `poetry run black .` )

Reply:
@sebastianneubauer added to `docs/source/install.md`